src/rpcblockchain.cpp

```diff
@@ -257,6 +257,7 @@ UniValue blockToJSON(const CBlock& block, const CBlockIndex* blockindex, bool fP
         result.pushKV("superblock", SuperblockToJson(block.GetSuperblock()));
     }
 
+    result.pushKV("fees_collected", RoundToString(GetFeesCollected(block) / (double)COIN, 8));
```

Comment:
Please use `ValueFromAmount()` for JSON fields that output money values. This will produce the correct formatting for a JSON number (not a string) that represents units of GRC. `RoundToString()` really needs to die 🙂

Reply:
done